src/util/specific.py

```diff
@@ -81,13 +81,13 @@ def rename_custom_model(cr, model_name, new_model_name, custom_module=None, repo
     )
 
 
-def rename_custom_module(cr, old_module_name, new_module_name, report_details=""):
-    cr.execute("SELECT 1 FROM ir_module_module WHERE name = %s", [old_module_name])
+def rename_custom_module(cr, old_module_name, new_module_name, report_details="", author="%"):
+    cr.execute("SELECT 1 FROM ir_module_module WHERE name = %s AND author ILIKE %s", [old_module_name, author])
     if not cr.rowcount:
-        _logger.warning("Module %r not found: skip renaming", old_module_name)
         return
 
     rename_module(cr, old_module_name, new_module_name)
+    _logger.warning("Custom module %r renamed to %r", old_module_name, new_module_name)
```

Comment:
Since the util is doing what we instructed it to do it should be just info.
```suggestion
    _logger.info("Custom module %r renamed to %r", old_module_name, new_module_name)
```
... unless you want to transform this into something meant for generic upgrade which seem to be the root issue in the discussion.

Reply:
> I think the warning level is appropriate here because there should be an inherent level of caution associated with renaming non-standard module.

We don't agree. For other reasons. There is always an inherent level of warning for almost everything we do in upgrades.

> IMO this shouldn't be addressed at specific or generic scripts. This should be the recipe to rename a custom module, period.

That seems like the goal of this PR. It was what I was referring above.